forge.config.js

```diff
@@ -17,9 +17,15 @@ module.exports = {
   makers: [
     {
       name: '@electron-forge/maker-squirrel',
-      config: {
-        name: 'signer'
-      }
+      config: process.env.SHOULD_SIGN
```

Comment:
Wouldn't this work?
`{ name: 'signer', ...(process.env.SHOULD_SIGN ? { certificateFile: bla } : {}) }`

Reply:
Is the conditional needed at all? Wouldn't it just ignore it if `WINDOWS_PFX_FILE` is `undefined`?

Reply:
🤷‍♂️

Reply:
Yes you're right @Mrtenz !